pkgs/tools/text/ocrmypdf/default.nix

```diff
@@ -45,6 +45,10 @@ in buildPythonApplication rec {
     setuptools_scm
   ];
 
+  propagatedNativeBuildInputs = with python3Packages; [
+    setuptools
```

Comment:
I believe this can be removed, I tested it locally, and was able to get it to build and print a usage statement